ext_mongodb.php

```diff
@@ -19,6 +19,108 @@ function bsonUnserialize(array $data) : void;
 {
 }
 
+/* == MONITORING ============================================================= */
+namespace MongoDB\Driver\Monitoring;
+
+/* ------ Building blocks for value obejcts */
+abstract class CommandEvent
+{
+	private function __construct()
+	{
+		throw new Exception\RunTimeException("Accessing private constructor");
+	}
+
+	public function getCommandName() : string;
+
+	// do we need this? I don't think so, as you pass in an object
+	public function getContext() : mixed;
+
+	// libmongoc use getHost, but spec says "connectionId".
+	public function getHost() : \MongoDB\Driver\Server;
```

Comment:
I'd be in favor of calling this `getServer()` if we'll be returning our own server object.